cmd/traffic/cmd/manager/internal/cluster/info.go

```diff
@@ -287,27 +293,28 @@ func toRPCSubnets(cidrMap subnet.Set) []*rpc.IPNet {
 
 // GetTrafficAgentPods gets all pods that have a `traffic-agent` container
 // in them.
-func (oi *info) GetTrafficAgentPods(ctx context.Context, agents string) ([]*kates.Pod, error) {
-	client := managerutil.GetKatesClient(ctx)
-	var pods []*kates.Pod
+func (oi *info) GetTrafficAgentPods(ctx context.Context, agents string) ([]*corev1.Pod, error) {
+	informerFactory := informers.NewSharedInformerFactory(managerutil.GetK8sClientset(ctx), 0)
+	podController := informerFactory.Core().V1().Pods()
+	podLister := podController.Lister()
+	pods, err := podLister.List(labels.Everything())
+	if err != nil {
+		return nil, err
+	}
 
 	// We don't get agents if they explicitly say false
-	if agents == "False" {
+	if agents == "None" {
 		return nil, nil
 	}
-	if err := client.List(ctx, kates.Query{Kind: "Pod"}, &pods); err != nil {
-		return nil, err
-	}
-	var agentPods []*kates.Pod
+	var agentPods []*corev1.Pod
 	for _, pod := range pods {
-		if agents != "all" {
-			if !strings.Contains(pod.Name, agents) {
-				continue
-			}
+		if agents != "all" && !strings.Contains(pod.Name, agents) {
+			continue
 		}
 		for _, container := range pod.Spec.Containers {
 			if container.Name == "traffic-agent" {
```

Comment:
pls use `install.AgentContainerName` instead of the magic string

Reply:
good callout, will also use `install.ManagerAppName` for the `traffic-manager`